CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## Unreleased
+
+* The undocumented `pyi.__version__` and `pyi.PyiTreeChecker.version`
+  attributes has been removed. Use `flake8 --version` from the command line, or
+  `importlib.metadata.version("flake8_pyi")` at runtime, to determine the
+  version of `flake8-pyi` installed at runtime.
```

Comment:
Possibly we could add deprecation warnings for these, and remove them in a later release. But honestly, I think it's exceedingly unlikely that anybody's using them currently.